code-challenges.test.js

```diff
@@ -17,28 +17,70 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var num1 = 15
-// Expected output: "15 is divisible by three"
 
-var num2 = 0
+describe("when evenlyDivisible is called"); => {
+
+  it("returns '15 is divisble by three when called with 15'", () => {
+    //arrange
+    const argument = 15
+    //act
+    const actualDivisable = evenlyDivisible(argument);
+    //assert
+    expect(actualDivisable).toequal("15 is divisable by three")
+  });
+//Expected output: "15 is divisible by three"
+
+  it("returns '0 is divisble by three' when called with 0", () => {
+    //arrange
+    const argument = 0
+    //act
+    const actualDivisable = evenlyDivisible(argument);
+    //assert
+    expect(actualDivisable).toequal("0 is divisble by three");
+  });
 // Expected output: "0 is divisible by three"
 
-var num3 = -7
-// Expected output: "-7 is not divisble by three"
+  it("returns '-7 is not divisable by 3' when called with -7", () => {
+    //arrange
+    const argument = -7
+    //act
+    const actualDivisable = evenlyDivisible(argument);
+    //assert
+    expect(actualDivisable).toequal("-7 is divisable by three");
+  });
 
+// Expected output: "-7 is not divisble by three"
+// b) Create the function that makes the test pass.
+var num1 = 15
+var num2 = 0
+var num3 = -7
 
 
+if(num1 % 2) {
+  return ("15 is divisble by 3")
+}
+else if (num2 % 2) {
```

Comment:
```suggestion
else if (num2 % 3) {
```
You'd want to divide by 3 to check if a number is divisible by 3. In using the modulo operator, something like `5 % 3` will evaluate to `2` because there is the remainder when 5 is divided by 3. And then check to see if the remainder is 0 or not 0 to determine if it's divisible.